modules/core-services/services/member/member.go

```diff
@@ -210,6 +219,17 @@ func (m *Member) CreateOrUpdate(userID string, req apistructs.MemberAddRequest)
 			// 创建成员
 			if len(members) == 0 {
 				orgID, projectID, applicationID := m.getIDs(req, targetScopeID)
+				if req.Scope.Type == apistructs.OrgScope {
```

Comment:
better add a TODO comment, we will not auto create PAT when user invited to org member, after we finish the page for PAT management.

Reply:
done